cfgov/research_reports/jinja2/index.html

```diff
@@ -0,0 +1,114 @@
+{% set is_report = True %}
+{% extends 'layout-side-nav.html' %}
+
+{% import './pdf-download.html' as pdf_download with context %}
+{% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
+
+{% block css %}
+{{ super() }}
+<link rel="stylesheet" href="{{ static('apps/research-reports/css/main.css') }}">
+{% endblock %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom research-report
+{%- endblock %}
+
+{% if report_sections is not defined  %}
+    {% set report_sections = get_report_sections(request, page) %}
+{% endif %}
+
+{% if report_appendices is not defined  %}
+    {% set report_appendices = get_report_appendices(request, page) %}
+{% endif %}
+
+{% set researchers = get_researchers() %}
+
+
+{% macro report_part(sections, is_appendix) %}
+  {% for section in sections -%}
+    <h2 class="report-header">
+      <a
+        id="{{section.url[1:]}}"
```

Comment:
What's the motivation for the `[1:]` here?

Reply:
The `section.url` variable always starts with a `#` which we don't want to pass through to the id. Perhaps this would be more cogent if the variable name was `fragment`?

Reply:
Yes, it might be clearer if the `url` variable had a different name (e.g. `section_id`) and was defined without the hash, and then the hash gets appended where it needs to be?